.travis.yml

```diff
@@ -29,3 +29,15 @@ deploy:
   on:
     repo: concord-consortium/populations.js
     branch: production
+- provider: s3
+  access_key_id:
+    secure: X48QoG+ZgSck1n3+VSAR/lzKF65cxPEdbbp8TcKAHaBDfRh5twSowbE0QE8l9yE4UbRSkmtbSCmsGkxsi6+AX4UZ3FkHR9Ke93ZHua1ncdfSOS+BMtfUl7/Q+Yhp8619G6VLGYDFQEmip89pNi+F5aFud9X9+qObWFJC1haVXB8=
+  secret_access_key:
+    secure: USiMwhfwJO2tEbQODWBUqjv5jwEfSFxn6v0KcaPdxNVLCDrXPFvmIBgP7F90lr5ulSMwM2d8deb2EMHkh7fCK2ABK+Qcvabez1VAjIdnKy7/1GROofxeeMBivosj8YEz+X9SJmdw7tJ4hAbkdcdIB0n9kW5uSGUVO0jl39zsknY=
```

Comment:
As @dstrawberrygirl pointed out in Slack, it would be good to eliminate the public display of keys, but since it's a preexisting condition that needn't hold up the PR.

Reply:
:+1: Deleted the file, but not looking into rewriting history now, as file has been around for years.